src/test/files/Project-with-library.ts

```diff
@@ -0,0 +1,1322 @@
+const originalProject: any = {
+  '_id': '5a0972db8833570d4b92471a',
+  'updatedAt': '2017-11-13T10:28:14.037Z',
+  'createdAt': '2017-11-13T10:24:27.978Z',
+  'ms3_version': '1.0.2',
+  '__v': 0,
+  '_creator': '587620f13b4e9f19192177e4',
+  'lockedBy': 'alexei',
+  '_deleted': false,
+  'examples': [
+    {
+      'title': 'ProjectExample',
+      'description': '',
+      'content': '<name>My Name<\/name>',
+      'format': 'xml',
+      'annotations': [
```

Comment:
you should test on already sanitized project, so remove any empty arrays, objects and strings